src/amphtml-email/10_Introduction/_Hello_World.html

```diff
@@ -65,17 +65,15 @@
       with equivalent or slightly enhanced custom AMP HTML tags (see
       [HTML Tags in the specification](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md)).
 
-      <p class="mb2">
         <div class="p1 ampstart-card abe-info"><strong>Important:</strong> URLs most use absolut paths in AMPHTML email.</div>
-      </p>
     -->
     <div class="images">
       <amp-img src="https://ampbyexample.com/img/amp.jpg" width="800" height="600" layout="responsive"></amp-img>
       <!--
         ### AMP Components
 
         The following AMP components are supported in AMPHTML email messages.
-        <h4>Dynamic Content </h4>
+        **Dynamic Content </h4>
```

Comment:
I missed this: remove the closing h4

Reply:
this is broken anyway and needs a complete refactor...plan to do this in another PR